docs/get-started/dapps/reach.md

```diff
@@ -0,0 +1,446 @@
+```
+
+Confirm the download worked by running
+
+
+``` bash
+$ ./reach version
+```
+
+
+Since Reach is Dockerized, when first used, the images it uses need to be downloaded. This will happen automatically when used for the first time, but can be done manually now by running
+
+
+```
+  $ ./reach update
+```
+
+
+You’ll know that everything is in order if you can run
+
+
+```
+  $ ./reach compile --help
+```
+
+
+To determine the current version is installed, run
+
+
+```
+  $ ./reach hashes
+```
+
+
+Output should look similar to:
+
+```
+reach: fb449c94
+reach-cli: fb449c94
+react-runner: fb449c94
+rpc-server: fb449c94
+runner: fb449c94
+devnet-algo: fb449c94
+devnet-cfx: fb449c94
+devnet-eth: fb449c94
+```
+
+
+
+All of the hashes listed should be the same and then visit the #releases channel on the [Reach Discord Server](https://discord.gg/9kbHPfwbwn) to see the current hashes.
+
+More information: Detailed Reach install instructions can be found in the [reach docs](https://reach.sh/). 
+
+## Clone the Reach Auction demo application
+
+Clone the repository using the following commands. 
+
+
+```bash
+// git clone algorand/reach-auction 
+```
+
+
+
+## Setup environments and run tests
+
+Navigate to your project folder
+
+```
+cd  ~/reach/auction
+```
+
+Sometimes it may be convenient to use the reach run command, preceded by setting the **REACH_CONNECTOR_MODE **, especially when testing multiple blockchain deployments.
+
+```
+  REACH_CONNECTOR_MODE=ALGO-devnet ./reach run
+```
+
+
+Set an environment variable to use the Algorand Blockchain. 
+
+
+```
+export REACH_CONNECTOR_MODE="ALGO-devnet"
+```
+
+
+# Application OverView
+
+
+## Alice and Bob’s auction design
+
+The sample app shown below is an auction. Alice and Bob think through the features for their dApp. They list off the following requirements:
+
+
+
+1. Sellers must be able to create a new auction for each piece of artwork. The artwork must be held by the contract after the auction begins and until the auction closes.
+2. The auction can be closed before it begins, in which case the artwork will be returned to the seller.
+3. Sellers can specify a reserve price, which if not met, will return the artwork to them.
+4. For each bid, if the new bid is higher than the previous bid, the previous bid will be refunded to the previous bidder and the new bid will be recorded and held by the contract.
+5. At the end of a successful auction, where the reserve price was met, the highest bidder will receive the artwork and the seller will receive the full bid amount.
+
+## The frontend and backend files that compose the Reach dApp.
+
+Copy these two files if not done already, into your  `~/reach/auction folder. `
+
+
+[index.rsh](https://github.com/algorand/reach-auction/blob/main/index.rsh)  is the Backend which provides the implementation of the solution in. It also determines what is published to the blockchain and how. It also defines the interfaces to the frontend.
```

Comment:
Are we sure this content will live on the algorand organization or algorand-devrel?

Reply:
Yes, the auction pyteal code is on Algorand as well and this is the counterpart for reach. Verified w Jason to leave under algorand